Additions/UIView-KIFAdditions.m

```diff
@@ -236,8 +253,17 @@ - (UIAccessibilityElement *)accessibilityElementMatchingBlock:(BOOL(^)(UIAccessi
                 if (indexPath) {
                     [indexPathsForVisibleRows addObject:indexPath];
                 }
+                if (firstIndexPath == nil) {
+                    firstIndexPath = indexPath;
+                } else {
+                    if ([firstIndexPath compare:indexPath] == NSOrderedDescending) {
+                        firstIndexPath = indexPath;
+                    }
+                }
             }];
-            
+
+			BOOL animationEnabled = [KIFUITestActor testActorAnimationsEnabled];
```

Comment:
NIT (since you need to push again): Spaces are used consistently throughout the file, please match.